27-cellular-connected-electronic-kiosk/README.md

```diff
@@ -0,0 +1,138 @@
+# Cellular-Connected Electronic Kiosk
+
+A cellular-based solution for downloading resources for an electronic kiosk display without an Internet connected, using a simple Python script.
+
+https://github.com/blues/app-accelerators/assets/20400845/04f94b74-fea8-4829-b673-e29d8782cf8c
```

Comment:
This would be more compelling if it were a video of the hardware showing/updating the screen, and not just the screen itself.

Reply:
I'll see if I can capture on my laptop - good suggestion